wallet/vault/vault.go

```diff
@@ -232,6 +232,21 @@ func (v *Vault) AllValidatorAddresses() []AddressInfo {
 	return addrs
 }
 
+func (v *Vault) AllAccountAddresses() []AddressInfo {
+	addrs := make([]AddressInfo, 0, v.AddressCount()/2)
+	for _, addrInfo := range v.Addresses {
+		addrPath, _ := addresspath.NewPathFromString(addrInfo.Path)
+		if addrPath.AddressType() == H(crypto.AddressTypeBLSAccount) {
```

Comment:
```suggestion
		if addrPath.AddressType() != H(crypto.AddressTypeValidator) {
```

This is more accurate. In future we may support other curves.

Reply:
Add a test for this function, in that tests the return addresses has no validator address